product_docs/docs/pgd/5/reference/conflict_functions.mdx

```diff
@@ -108,10 +108,10 @@ The view `bdr.node_log_config` shows all the logging configurations. It lists th
 
 If `log_to_table` is set to true, conflicts are logged to a table. The target table for conflict logging is `bdr.conflict_history`.
 
-This table is range partitioned on the column `local_time`. The table is managed by autopartition. By default, a new partition is created for every day, and conflicts of the last one month are maintained. After that, the old partitions are dropped automatically. Autopartition creates between 7 and 14 partitions in advance. bdr_superuser can change these defaults.
+This table is range partitioned on the column `local_time`. The table is managed by autopartition. By default, a new partition is created for every day, and conflicts of the last one month are maintained. After that, the old partitions are dropped. Autopartition creates between 7 and 14 partitions in advance. bdr_superuser can change these defaults.
 
 Since conflicts generated for all tables managed by PGD are logged to this table, it's important to ensure that only legitimate users can read the conflicted data. PGD does this by defining ROW LEVEL SECURITY policies on the `bdr.conflict_history` table. Only owners of the tables are allowed to read conflicts on the respective tables. If the underlying tables have RLS policies defined, enabled, and enforced, then even owners can't read the conflicts. RLS policies created with the FORCE option also apply to owners of the table. In that case, some or all rows in the underlying table might not be readable even to the owner. So PGD also enforces a stricter policy on the conflict log table.
 
-The default role `bdr_read_all_conflicts` can be granted to users who need to see all conflict details logged to the `bdr.conflict_history` table without also granting them `bdr_superuser` role.
+The default role bdr_read_all_conflicts can be granted to users who need to see all conflict details logged to the `bdr.conflict_history` table without also granting them bdr_superuser role.
```

Comment:
I am not sure that I agree with this edit. In my head, I would like to mark the role name with backticks to emphasize that we are dealing with a value that can be assigned. The role, like a table or view, is a resource that I would emphasize to separate from other terms. 

Any thoughts from others? @jpe442 @djw-m

Reply:
We don't tag role names.

Reply:
We need to be able to delineate role names, and as they are most typically typed, it seems reasonable to tag them with code tags. It's not like the text is saturated with them.

What's more concerning is the "The default role" which should be "The predefined role [bdr_read_all_conflicts](/security/pgd-predefined-roles/#bdr_read_all_conflicts)..."

and using a link provides the delineation in this case.

Reply:
```suggestion
The predefined role [bdr_read_all_conflicts](https://github.com/security/pgd-predefined-roles/#bdr_read_all_conflicts) can be granted to users who need to see all conflict details logged to the `bdr.conflict_history` table without also granting them bdr_superuser role.
```